docker/dev-local/prism_demo_config.js

```diff
@@ -0,0 +1,109 @@
+// Configuration file BC PRISM client app.
```

Comment:
This makes me nervous. We have two identical files in the repository, one for testing and one for production, it seems like it would be easy for them to get out of step and throw off testing. I don't have any better ideas than the setup here, though. Maybe the best we can do is to add a comment to this test version noting that it needs to be kept up to date with the other copy in the repository.

Reply:
This replicates the situation in deployment directories, which is what this really is, or is a template for. We could remove this file and  the container mount of it, but I very much wanted to preserve the record of how the configuration file is overridden, specifically the filepath.

Reply:
Makes sense.